### PR TITLE
Minor docs cleanup and homogenization.

### PR DESCRIPTION
**Pull request by lbakerchef**
Minor docs cleanup and homogenization.

### CODE REVIEW

File: docs-chef-io/content/server/upgrades.md [hunk 1]
```diff
@@ -211,6 +211,8 @@ Check the [post upgrade steps](#post-upgrade-steps) if you are upgrading from a
 
 ### External PostgreSQL
 
+External PostgreSQL instructions are provided as a courtesy only.
```
Comment (IanMadd) [inline docs-chef-io/content/server/upgrades.md:214]: I don't think we need this.
```suggestion
```

File: docs-chef-io/content/server/upgrades.md [hunk 2]
```diff
@@ -277,34 +279,40 @@ Check the [post upgrade steps](#post-upgrade-steps) if you are upgrading from a
 
    After performing the stepped upgrade, return here and continue with the next step below.
 
-1. [Download](https://downloads.chef.io/tools/infra-server) the Chef Infra Server version 14.8.X upgrade package.
+1. Download the desired Chef Infra Server version from the [Chef Infra Server Downloads](https://downloads.chef.io/products/chef-server).
```
Comment (IanMadd) [inline docs-chef-io/content/server/upgrades.md:282]:
```suggestion
1. [Download](https://downloads.chef.io/tools/infra-server) the desired version of Chef Infra Server.
```

File: docs-chef-io/content/server/upgrades.md [hunk 3]
```diff
@@ -277,34 +279,40 @@ Check the [post upgrade steps](#post-upgrade-steps) if you are upgrading from a
...
-   RPM:
+   To install with the RPM Package Manager:
 
    ```bash
-   sudo rpm -U /path/to/chef-server-core-VERSION.rpm
+   rpm -Uvh --nopostun --force /path/to/chef-server-core-VERSION.rpm
    ```
 
-1. Upgrade Chef Infra Server.
+1. Upgrade the server and accept the Chef Software license by entering `Yes` at the prompt:
```
Comment (IanMadd) [inline docs-chef-io/content/server/upgrades.md:304]:
```suggestion
1. Upgrade Chef Infra Server and accept the Chef Software license by entering `Yes` at the prompt:
```

File: docs-chef-io/content/server/upgrades.md [hunk 4]
```diff
@@ -277,34 +279,40 @@ Check the [post upgrade steps](#post-upgrade-steps) if you are upgrading from a
...
    ```bash
-   sudo CHEF_LICENSE='accept' chef-server-ctl upgrade
+   chef-server-ctl upgrade
    ```
 
+   To accept the license and upgrade in one command:
+
+   ```bash
+   CHEF_LICENSE='accept' chef-server-ctl upgrade
+    ```
```
Comment (IanMadd) [inline docs-chef-io/content/server/upgrades.md:314]:
````suggestion
   ```
````

File: docs-chef-io/content/server/upgrades.md [hunk 5]
```diff
@@ -393,7 +405,8 @@ Check the [post upgrade steps](#post-upgrade-steps) if you are upgrading from a
    sudo systemctl start postgresql.service
    ```
 
-1. Log in as the postgres user and confirm that the new PostgreSQL version is 13.3.
+1. Log in as the postgres user and confirm that the new PostgreSQL version is correct.
```
Comment (IanMadd) [inline docs-chef-io/content/server/upgrades.md:408]:
```suggestion
1. Log in as the Postgres user and confirm that the new PostgreSQL version is correct.
```

File: docs-chef-io/content/server/upgrades.md (hunk 1 above)
Comment (kagarmoe) [reply docs-chef-io/content/server/upgrades.md:214]: See the [PostgreSQL documentation](https://www.postgresql.org/docs/) for more information.

File: docs-chef-io/content/server/upgrades.md (hunk 5 above)
Comment (kagarmoe) [reply docs-chef-io/content/server/upgrades.md:408]: "Customarily, this user will be named postgres" from https://www.postgresql.org/docs/8.0/user-manag.html

I think we want:

```
Log in as the `postgres` user and confirm that the new PostgreSQL version is correct.
```

File: docs-chef-io/content/server/upgrades.md (hunk 1 above)
Comment (lbakerchef) [reply docs-chef-io/content/server/upgrades.md:214]: Yup.  I had previously thought that external postgresql people were 'on their own' so I didn't want us to assume any liability.  Ran it by someone else and that wasn't their impression.

File: docs-chef-io/content/server/upgrades.md (hunk 5 above)
Comment (lbakerchef) [reply docs-chef-io/content/server/upgrades.md:407]: Whoops, good catch.

Comment (lbakerchef) [reply docs-chef-io/content/server/upgrades.md]: Caught another one also.

File: docs-chef-io/content/server/upgrades.md (hunk 4 above)
Comment (lbakerchef) [reply docs-chef-io/content/server/upgrades.md]: Good eyes.  Caught another one also.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**